ci/tasks/helpers.sh

```diff
@@ -4,7 +4,7 @@ export MODULES_GIT_LONG_REF="$(cat modules/.git/ref)"
 export KUBE_CONFIG="~/.kube/config"
 export CI_ROOT="$(pwd)"
 export CI_ROOT_DIR="${CI_ROOT##*/}"
-export TF_VAR_name_prefix="testflight-$(cat testflight-uid/version | tr . -)"
+export TF_VAR_name_prefix="testflight$(cat testflight-uid/version | tr -d .)"
```

Comment:
I had done this in order to get rid of `-` in the name prefix because many resources can't have hyphens on the name, but now, I think the module should handle this itself because the name prefix can have hyphens I guess - thoughts? @bodymindarts @krtk6160

Reply:
Maybe @fireballpoint1 this is for you